backend/pkg/rest/server.go

```diff
@@ -109,15 +96,19 @@ func (s *Server) GetNamespaceList() ([]string, error) {
 
 func CreateRESTServer(port int, dbHandler *database.Handler, scanner orchestrator.VulnerabilitiesScanner,
 	clientset kubernetes.Interface) (*Server, error) {
+	scanChan := make(chan *runtime_scanner.ScanConfig, 10) // TODO what size of channels?
```

Comment:
I think we should not state a size since only one scan at a time is allowed, DYT?

Reply:
Regarding the results channel, probably yes. Regarding the scan channel - no guraunty that I will not get a quick scan while handling schedule scan, maybe better to set it to 2?

Reply:
we should not allow quick scan during schedule scan, it will not work. I hope that using this channel (with no buffer) will prevent it, but the UI should know that there is a scan in progress and not allow new start scan before cancelling the previous.

Reply:
I see that `startScan` is starting a scan and waiting for scan done in the background while returning back to watch on the scan channel, why you implemented it like that? This will allow starting a scan while a previous one already started without calling the `Stop` API.